src/AccountActionsHistory.js

```diff
@@ -373,22 +440,13 @@ class AccountActionsHistory extends React.Component {
             id="list-accountactions"
             formatter={accountActionsFormatter}
             columnMapping={columnMapping}
-            visibleColumns={[
-              'date',
-              'action',
-              'amount',
-              'balance',
-              'transactioninfo',
-              'created',
-              'source',
-              'comments',
-            ]}
+            visibleColumns={columns}
             contentData={(account.id === (actions[0] || {}).accountId) ? actionsSort : []}
```

Comment:
I'm not sure that it's a good way of making conditions. Here we are creating an empty object, which obviously won't have the `accountId` property. All further comparisons do not make sense. Maybe it's better to rewrite it like this:
```suggestion
    let contentData = [];
    if (actions[0] && actions[0].accountId === account.id) {
      contentData = actionsSort
    }
```

More lines, but it's easier to read and hos no unneeded steps. What do you think?